docs/docs/writing-documentation-with-docz.md

```diff
@@ -0,0 +1,154 @@
+
+```shell
+npm install --save-dev docz docz-theme-default docz-plugin-css @babel/plugin-syntax-dynamic-import webpack-merge
+```
+
+Add the following scripts to your `package.json` file:
+
+```json:package.json
+{
+    ...
+    "scripts": {
+        ...
+        "docz:dev": "docz dev",
+        "docz:build": "docz build",
+        ...
+    }
+    ...
+}
+```
+
+We are now going to create two files:
+
+- `doczrc.js` to configure Docz,
+- `docz/wrapper.js` to inject some JavaScript in Docz pages, to ensure compatibility with Gatsby.
+
+Create a new folder `docz`, and inside that folder, a new file `wrapper.js`. Copy and paste there the following code:
+
+```js:title=docz/wrapper.js
+import * as React from "react"
+
+// Gatsby's Link overrides:
+// Gatsby internal mocking to prevent unnecessary error in docz: __PATH_PREFIX__ is not defined
+global.__PATH_PREFIX__ = ""
+
+export default ({ children }) => children
+```
+
+> We are essentially creating a dummy wrapper that does nothing else than making sure that `global.__PATH_PREFIX__` is defined on every page.
+
+Create now a new file called `doczrc.js` in the root directory of your Gatsby project, and add the following content:
+
+```js:title=doczrc.js
+import merge from "webpack-merge"
+import { css } from "docz-plugin-css"
+
+export default {
+  title: "Docz with Gatsby",
+  // Add CSS support in case you use them in your Gatsby project
+  plugins: [css()],
+  // highlight-next-line
+  // Wrapper used to inject some global variable mocks
+  wrapper: "docz/wrapper.js",
+  modifyBundlerConfig: config => {
+    const gatsbyNecessaryConfig = {
+      module: {
+        rules: [
+          {
+            // Transpile Gatsby module because Gatsby includes un-transpiled ES6 code.
+            // Ignore .json files because they fail to be parsed
+            exclude: [/node_modules\/(?!(gatsby)\/)/, /\.json$/],
+            use: [
+              {
+                // use installed babel-loader which is v8.0-beta (which is meant to work with @babel/core@7)
+                loader: "babel-loader",
+                options: {
+                  // use @babel/preset-react for JSX and env (instead of staged presets)
+                  presets: ["@babel/preset-react", "@babel/preset-env"],
+                  plugins: [
+                    // use @babel/plugin-proposal-class-properties for class arrow functions
+                    "@babel/plugin-proposal-class-properties",
+                    // use @babel/plugin-syntax-dynamic-import for dynamic import support
+                    "@babel/plugin-syntax-dynamic-import",
+                  ],
+                },
+              },
+            ],
+          },
+        ],
+      },
+    }
+
+    return merge(gatsbyNecessaryConfig, config)
+  },
+}
+```
+
+Once you have this configured you should run Docz to ensure it can start up properly. You should see by default a _Page Not Found_ page: we haven't created any `mdx` file yet. To run Docz:
```

Comment:
```suggestion
Once you have this configured you should run Docz to ensure it can start up properly. You should see by default a _Page Not Found_ page: You haven't created any `mdx` file yet. To run Docz:
```